test/e2e/e2e_test.go

```diff
@@ -84,7 +84,9 @@ var _ = Describe("Workload cluster creation", func() {
 			test_remediation()
 			pivoting()
 			cert_rotation()
+			upgrade_ironic()
```

Comment:
nit: I know this is consistent with the other test names, but is there a reason we're using snake_case instead of camelCase here? I wonder if we should follow up and make these camelCase.